frontend/src/Login.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { GOOGLE_AUTH_URL } from './Constants';
+import brevityLogo from './img/brevity-logo.png';
+import GoogleButton from 'react-google-button';
+import './Login.css';
+import { Container, Image, Row, Col } from 'react-bootstrap';
+
+class Login extends React.Component {
+    constructor(props) {
+        super(props);
+        this.props = props;
+    }
+
+
+    render() {
+        return(
+            <Container className="login-container">
+                <Row className="login-logo">
+                    <Col></Col>
+                    <Col md={6}>
```

Comment:
You can change this to `md={{span: 6, offset: 3}}` to get rid of the empty `<Col></Col>`s